### PR TITLE
Fix slack & community link

### LINKED ISSUE #93: The slack Logo is not redirecting to the slack link but to the meshery.io/community page
#### Current Behavior

The slack Logo is not redirecting to the slack link but to the meshery.io/community page.

#### Expected Behavior

Interchange both the links (Slack & Community) in the README.md file.

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 📝 [Meshery Adapters Spreadsheet](https://docs.google.com/spreadsheets/d/1rGGpSXC68iDJzNRU-qZIRQsZUwkt1qLFg7JCs4pfvbU/edit?usp=sharing)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by RahulGopathi**
Signed-off-by: RahulGopathi <gopathi.1@iitj.ac.in>

**Description**

This PR fixes #93 

The slack & community link have been interchanged and it is now correct in README.md

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by RahulGopathi** [comment]
@UtkarshMishra12 please have a look and suggest changes if any ;)

File: README.md [hunk 1]
```diff
@@ -62,9 +62,9 @@ Find out more on the <a href="https://layer5.io/community">Layer5 community</a>.
 
 <div>&nbsp;</div>
 
-<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src="https://raw.githubusercontent.com/meshery/meshery-cilium/master/.github/readme/images//slack-128.png" style="margin-left:10px;padding-top:5px;" width="110px" align="right" /></a>
+<a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src="https://raw.githubusercontent.com/meshery/meshery-cilium/master/.github/readme/images//slack-128.png" style="margin-left:10px;padding-top:5px;" width="110px" align="right" /></a>
```
Comment (UtkarshMishra12) [inline README.md:65]: Can you please make it https rather than http?

**Review by UtkarshMishra12 — CHANGES_REQUESTED** [review]

**Review by UtkarshMishra12 — APPROVED** [review]
LGTM! Nice work @RahulGopathi 😎

**Comment by leecalcote** [comment]
Thank you, gents.

**Pull request merged**